tests/resources/yahoo_samples/rebuild_samples.py

```diff
@@ -0,0 +1,78 @@
+"""
+Script for rebuilding the samples for the Yahoo tests.
+"""
+from textwrap import dedent
+
+import numpy as np
+import pandas as pd
+
+from zipline.testing import test_resource_path, write_compressed
+from zipline.utils.tradingcalendar import trading_days
+
+
+def zipfile_path(symbol, ext):
+    return test_resource_path('yahoo_samples', symbol + ext + '.gz')
+
+
+def pricing_for_sid(sid):
+    modifier = {
+        'Low': 0,
+        'Open': 1,
+        'Close': 2,
+        'High': 3,
+        'Volume': 0,
+    }
+
+    def c(name):
```

Comment:
What does "c" stand for here?  In the previous module, it stood for "cache".  Could we use longer names?